packages/cli/src/commands/org/create.ts

```diff
@@ -0,0 +1,50 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {Config} from '../../lib/config/config';
+import {
+  Preconditions,
+  HasAccessToken,
+} from '../../lib/decorators/preconditions/';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+
+export default class OrgCreate extends Command {
+  static description = 'Create a new trial Coveo organization';
+
+  private configuration!: Config;
+
+  static flags = {
+    setDefaultOrganization: flags.boolean({
+      char: 'D',
```

Comment:
Lowercase `d`, similar to all other flags we use for other commands.

Reply:
so lower case `O` too?

Reply:
Also, this is our first flag with multiple words, did we look at the best practices?

Reply:
Herp derp, je viens de catcher.